### ISSUE TITLE
Create a tutorial on how to integrate IPFS as the file system for Angular applications.

### ISSUE THREAD

**Issue by shaungt1** [ipfs/ipfs-docs#537]
I would like to know if there are more instructions as to set up a website (Angular, Vue, React..) and use IPFS to host images, files, and or user data. Allowing an SPA to be completely decentralized or semi-decentralized

**Comment by johnnymatthews** [comment]
Such tutorials do not yet exist, but we're looking to create more web-developer focused guides and docs in 2021. Do you have any specific topics you'd like to see discussed? This will help us decide what to build first.

**Comment by shaungt1** [comment]
Yes I would like to know how to integrate IPFS as the file system for Angular applications. More so the logistics such as if I would need to separately host  the web app or  if hosting an Angular SPA on IPFS a viable option? Maybe more novice understanding 
 of how a web apps such as DTube are implemented on IPFS (Not Steam Chain Only IPFS).

 For instance I have a SPA chat/image application I want to make fully free and decentralized I have a server build with 10 parallel Raspberry PI's I would like to set up the IPFS node with the web application to directly store Files. 

The Idea is to make an easy cross platform packaged FREE implementation that anyone can execute and install on there machine(Windows, Lunix, MacOS, RasPi, etc.) and accesses the browser be able to converse in real time with others nodes similar to Facebook. Only the images and data they post is completely  stored in thier IPFS node on thier machines. Not on some centralized server that exploited people for ad monetization.

 That is the goal, but I want to know some of the basics of using dynamic content, scripts, and applications and interacting inputs and outputs of IPFS in this nature?

The only reason I am using RasPi for development server so it can stay online 100% of the time, hence my IPFS node is always available.

regardless topics about basic interactions if the nature would great. I think it would help other developers who want to connect thier projects as well and self serve nodes and data while being heuristic.

**Comment by johnnymatthews** [comment]
Thanks for your input @shaungt1, it's really appreciated. I've changed the title of this issue to reflect the main actionable to come out of your request. Next, we need to figure out how this integration would work, and then create a tutorial based on those findings. There might be a bounty placed on this issue in early 2021, once we have fleshed out exactly what we need to do. Feel free the post more thoughts and findings in the meantime.
